smack-core/src/main/java/org/jivesoftware/smack/packet/MessageBuilder.java

```diff
@@ -53,8 +53,7 @@ private void copyFromMessage(Message message) {
 
     @Override
     protected void addStanzaSpecificAttributes(ToStringUtil.Builder builder) {
-        builder.addValue("type", type)
-               ;
+        builder.addValue("type", type);
```

Comment:
That's an unrelated change in this commit. Also commit subject could be "[core] Fix typo in MessageBuilder"

Reply:
> Such kind of change (obvious code style cleanup) is too small for a separate commit

It's not.